io.defn.Franz.yml

```diff
@@ -0,0 +1,27 @@
+app-id: io.defn.Franz
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: Franz
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=home
+  - --filesystem=/tmp
```

Comment:
Why?

Reply:
Lua scripts get written to `/tmp` before being executed.

Reply:
Will setting TMPDIR to somewhere inside the sandbox work or does it need to be host temp?

Reply:
You can set it with a wrapper over the binary or `--env=TMPDIR`, eg. `export TMPDIR=$XDG_CACHE_HOME/tmp`

Reply:
Thanks. It looks like everything still works when I drop the flag, no wrapping necessary.